### PR TITLE
Wedge bond from atrop error

### PR DESCRIPTION
**Pull request by tadhurst-cdd**
#### Reference Issue
fixed a bug in determining which bonds to wedge for an atropisomer

#### What does this implement/fix? Explain your changes.
If a bond had directions set to help determine the cis-trans config for a double bond, and also COULD be used to define an atropisomer bond, an error was thrown.  Now the error is not generated and the bond is simply skipped.

#### Any other comments?

Also made a change to help my build work having to do with string_view inside a catch2 macro.

### CODE REVIEW

File: Code/GraphMol/StructChecker/StripSmallFragments.cpp
Comment (greglandrum) [inline Code/GraphMol/StructChecker/StripSmallFragments.cpp:1]: This change is unrelated to the PR. Please remove it

**Review by greglandrum — CHANGES_REQUESTED** [review]
The changes which are related to the topic of PR look fine to me.
The two changes that are not related ought to be removed unless there's a reason behind them that I'm not seeing.

File: Code/GraphMol/FileParsers/file_parsers_catch.cpp [hunk 1]
```diff
@@ -6319,7 +6319,7 @@ TEST_CASE("MaeWriter basic testing", "[mae][MaeWriter][writer]") {
 
     std::string_view ctBlock(&mae[ctBlockStart]);
 
-    CHECK(ctBlock == MaeWriter::getText(*mol));
+    CHECK((ctBlock == MaeWriter::getText(*mol)) == true);
```
[the remark this answers is not in the document]
Comment (tadhurst-cdd) [reply Code/GraphMol/FileParsers/file_parsers_catch.cpp]: I will remove the change, but without it the system does not build.  There is a problem between string_view and Catch2.

File: Code/GraphMol/StructChecker/StripSmallFragments.cpp
Comment (tadhurst-cdd) [reply Code/GraphMol/StructChecker/StripSmallFragments.cpp:1]: Again, I have removed the change.  But is the structure checker flag is on for the build, this causes an error.

File: Code/GraphMol/FileParsers/file_parsers_catch.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (greglandrum) [reply Code/GraphMol/FileParsers/file_parsers_catch.cpp]: > I will remove the change, but without it the system does not build. There is a problem between string_view and Catch2.

This does not happen for me on any machine I build the RDKit on, nor does it happen on any of the CI systems. This is likely a problem with your local configuration

**Review by greglandrum — APPROVED** [review]
LGTM

File: Code/GraphMol/FileParsers/file_parsers_catch.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (DavidACosgrove) [reply Code/GraphMol/FileParsers/file_parsers_catch.cpp]: This happened to me also, on my Macs.  It seems that it is because Catch2 needs to be compiled with C++17 compatibility in order to use string_view and by default its build uses C++14.  I deleted my installation in /usr/local and RDKit built a version that worked for me.

**Comment by tadhurst-cdd** [comment]
I just made a separate PR for this change.  I think we are using the structure checker at CDD

 

Dr. Tad Hurst

Collaborative Drug Discovery, Inc.

 ***@***.***> ***@***.***

314-610-4313
